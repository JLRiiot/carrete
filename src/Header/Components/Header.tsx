import AppBar from '@material-ui/core/AppBar';
import IconButton from '@material-ui/core/IconButton';
import {
  createStyles,
  Theme,
  withStyles,
  WithStyles
  } from '@material-ui/core/styles';
import Toolbar from '@material-ui/core/Toolbar';
import MenuIcon from '@material-ui/icons/Menu';
import React, { Component } from 'react';
import BrowseField from '../../Browse/Containers/Browse';

const styles = (theme: Theme) =>
  createStyles({
    root: {
      width: '100%'
    },
    menuButton: {
      marginLeft: -12,
      marginRight: 20
    },
    title: {
      display: 'none',
      [theme.breakpoints.up('sm')]: {
        display: 'block'
      }
    }
  });

export interface HeaderProps extends WithStyles<typeof styles> {}
interface HeaderState {}

class Header extends Component<HeaderProps, HeaderState> {
  render() {
    const { classes } = this.props;
    return (
      <div className={classes.root}>
        <AppBar position="fixed">
          <Toolbar>
            <IconButton className={classes.menuButton} color="inherit" aria-label="Open drawer">
              <MenuIcon />
            </IconButton>
            <BrowseField />
          </Toolbar>
        </AppBar>
      </div>
    );
  }
}

export default withStyles(styles)(Header);
