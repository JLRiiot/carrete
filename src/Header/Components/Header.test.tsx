import Header from './Header';

it('renders the right components', () => {
  let wrapper = shallow(<Header />);

  expect(wrapper).toMatchSnapshot();
});
