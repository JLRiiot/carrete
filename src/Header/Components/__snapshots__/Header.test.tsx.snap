// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the right components 1`] = `
<Header
  classes={
    Object {
      "menuButton": "Header-menuButton-2",
      "root": "Header-root-1",
      "title": "Header-title-3",
    }
  }
/>
`;
